java-wrapper/src/main/java/com/codedisaster/steamworks/SteamNetworkingMessage.java

```diff
@@ -0,0 +1,22 @@
+package com.codedisaster.steamworks;
+
+import java.nio.ByteBuffer;
```

Comment:
I believe this import can just be removed, since it isn't used. You might want to run the "Optimize Imports" inspection on the 4 Java files you added. It shouldn't matter unless you import something that has a large static initializer, or something like that, but it might mess with jlink minimization if there are too many unused imports.

Reply:
I've gone ahead and updated the files to have correct imports.